frontend/app/profile/page.tsx

```diff
@@ -0,0 +1,367 @@
+"use client";
+import { Dialog, Transition } from "@headlessui/react";
+import Image from "next/image";
+import { Fragment, useState } from "react";
+import DP from "./../../public/images/Ashley & Owen.png";
+import eye from "./../../public/images/Eye.png";
+import location from "./../../public/images/location.png";
+import Upload from "./../../public/images/upload.png";
+
+const calculateAge = (birthday) => {
```

Comment:
birthday should be typed (to avoid vscode linting errors with the red squiggly line) e.g., ('birthday: string').